modules/web/src/app/shared/components/cluster-summary/template.html

```diff
@@ -385,7 +385,7 @@ <h4>IPv6</h4>
               <div value>{{cluster.spec.cloud?.azure.vnet}}</div>
             </km-property>
 
-            <km-property-boolean *ngIf="cluster.spec.cloud?.azure?.assignAvailabilitySet"
+            <km-property-boolean *ngIf="cluster.spec.cloud?.azure?.assignAvailabilitySet !== undefined"
```

Comment:
i think usually if the value is false we don't show it on the summary step so i think no need for this change